pt-BR/mail/chrome/messenger/phishing-afterload-warning-message.dtd

```diff
@@ -1,28 +1,28 @@
+<!ENTITY safeb.palm.message.p1.linkText "Leia mais em &#187;">
+<!ENTITY safeb.palm.p1.linkStatusText "Leia mais em &#133;">
 
-<!ENTITY safeb.palm.message.p2.start "These types of web forgeries are used in scams known as phishing attacks, in which fraudulent web pages and emails are used to imitate sources you may trust. You can find out more about ">
-<!ENTITY safeb.palm.message.p2.linkText "how &brandShortName; protects you">
-<!ENTITY safeb.palm.message.p2.end " from phishing attacks.">
+<!ENTITY safeb.palm.message.p2.start "Esses tipos de falsificações são utilizados em golpes conhecidos como ataques de phishing, em que páginas de Internet falsas e e-mails são utilizados para imitar fontes nas quais você confia. Você pode saber mais sobre ">
+<!ENTITY safeb.palm.message.p2.linkText "como o &brandShortName; te protege">
+<!ENTITY safeb.palm.message.p2.end " de ataques de phishing.">
 
-<!ENTITY safeb.palm.accept.label "Get me out of here!">
-<!ENTITY safeb.palm.accept.statustext "Navigate to my home page">
-<!ENTITY safeb.palm.decline.label "Ignore this warning">
-<!ENTITY safeb.palm.decline.statustext "Close warning" >
-<!ENTITY safeb.palm.notforgery.label2 "This isn't a web forgery…">
-<!ENTITY safeb.palm.report.label "Why was this site blocked?">
+<!ENTITY safeb.palm.accept.label "Me tire daqui!">
+<!ENTITY safeb.palm.accept.statustext "Navegar para minha página inicial">
+<!ENTITY safeb.palm.decline.label "Ignorar este aviso">
+<!ENTITY safeb.palm.decline.statustext "Fechar aviso" >
+<!ENTITY safeb.palm.notforgery.label2 "Não é uma falsificação…">
+<!ENTITY safeb.palm.report.label "Por que este site foi bloqueado?">
 
-<!ENTITY safeb.urlbaricon.tooltip "This page might be dangerous; click for details.">
+<!ENTITY safeb.urlbaricon.tooltip "Esta página pode ser perigosa. Clique para mais detalhes.">
 
-<!ENTITY safeb.blocked.malware.title "Reported Attack Site!">
+<!ENTITY safeb.blocked.malware.title "Site de ataque reportado!">
 <!-- Localization note (safeb.blocked.malware.shortDesc) - Please don't translate the contents of the <span id="malware_sitename"/> tag.  It will be replaced at runtime with a domain name (e.g. www.badsite.com) -->
-<!ENTITY safeb.blocked.malware.shortDesc "This web site at <span id='malware_sitename'/> has been reported as an attack site and has been blocked based on your security preferences.">
-<!ENTITY safeb.blocked.malware.longDesc "<p>Attack sites try to install programs that steal private information, use your computer to attack others, or damage your system.</p><p>Some attack sites intentionally distribute harmful software, but many are compromised without the knowledge or permission of their owners.</p>">
+<!ENTITY safeb.blocked.malware.shortDesc "O site em <span id='malware_sitename'/> foi reportado como um site que realiza ataques e foi bloqueado baseado nas suas preferências de segurança.">
```

Comment:
CORRECTIONS NEEDED it should be <!ENTITY safeb.blocked.malware.shortDesc "O site em <span id='malware_sitename'/> foi reportado como um site que realiza ataques e foi bloqueado, com base nas suas preferências de segurança.">